akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1236,27 +1254,6 @@ class DDataShardCoordinator(
         sendCoordinatorStateUpdate(evt)
       }
 
-    case UpdateSuccess(AllShardsKey, Some(newShard: String)) =>
-      log.debug("The coordinator shards state was successfully updated with {}", newShard)
-      val newRemainingKeys = remainingKeys - AllShardsKey
-      if (newRemainingKeys.isEmpty)
-        unbecomeAfterUpdate(evt, afterUpdateCallback)
-      else
-        context.become(waitingForUpdate(evt, afterUpdateCallback, newRemainingKeys))
-
-    case UpdateTimeout(AllShardsKey, Some(newShard: String)) =>
-      log.error(
-        "The ShardCoordinator was unable to update shards distributed state within 'updating-state-timeout': {} millis ({}), event={}",
-        writeMajority.timeout.toMillis,
-        if (terminating) "terminating" else "retrying",
-        evt)
-      if (terminating) {
-        context.stop(self)
```

Comment:
terminating not handled in same way?

Reply:
Good point. The timeout/retry was pushed into the remember entities provider and I didn't add a separate timeout here yet, need to think about whether that makes sense or not (and why waiting for this timeout before stopping makes sense).

Reply:
Do'h, I'm lying I did re-add it with the `RememberEntityTimeout`